src/components/Navbar.js

```diff
@@ -45,28 +65,31 @@ function Navbar() {
 									icon={faAngleDown}
 								/>
 							</button>
-							<ul className="dropdown-menu absolute hidden pt-2 bg-white shadow-2xl">
-								<Link to="/courses/python/beginner">
-									<li className="rounded-t py-2 px-4 block whitespace-no-wrap hover:bg-gray-200 transition-color duration-200">
-										Python Beginner
-									</li>
-								</Link>
-								<Link to="/courses/python/intermediate">
-									<li className="py-2 px-4 block whitespace-no-wrap hover:bg-gray-200 transition-color duration-200">
-										Python Intermediate
-									</li>
-								</Link>
-								<Link to="/courses/python/advanced">
-									<li className="rounded-b py-2 px-4 block whitespace-no-wrap hover:bg-gray-200 transition-color duration-200">
-										Python Advanced
-									</li>
-								</Link>
+
+							<ul className="dropdown-menu absolute hidden pt-2 bg-white shadow-2xl rounded-lg">
+								<li>
+									{courseItems.map((item, i) => {
+										return (
+											<Link
+												key={i}
+												to={item.to}
+												className="py-2 px-4 block whitespace-no-wrap hover:bg-gray-200 transition-color duration-200"
+											>
+												{item.display}
+											</Link>
+										);
+									})}
+								</li>
```

Comment:
This doesn't seem right... You're putting all the links in a single `<li>`.
Shouldn't you make a new `<li>` for each element?